subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dsl/DefaultComponentMetadataProcessor.java

```diff
@@ -360,16 +360,12 @@ public void addVariant(String name, String base, Action<? super VariantMetadata>
 
         @Override
         public void belongsTo(Object notation) {
-            belongsTo(notation, true);
+
```

Comment:
:o: I'm confused both `belongsTo` implementation are now no-op, can you explain?

Reply:
This is unrelated to the actual changes. It is a separate commit:

https://github.com/gradle/gradle/pull/11465/commits/7a11fd0c337d460f440970889fdd4a399bc17417

This was no-op code already. It was just filling a `owners` list but not doing anything with it. Probably a leftover from an earlier change. It also makes sense that we do not have to do anything here, because owners are also serialized (we don't need to recompute them in some way).

Reply:
I see, thanks!